### PR TITLE
updating_team

### PR DESCRIPTION
**Pull request by jcmsalves**
Almost time to say goodbye, removing my name from the list 😢 
It has been a massive pleasure being part of this team, thank you so much everyone.

giphy

### CODE REVIEW

**Review by JEKMiles — APPROVED** [review]

**Review by kioba — APPROVED** [review]

**Comment by iTomkinas** [comment]
Will I get skyscanner discounts? 😄

File: README.md [hunk 1]
```diff
@@ -28,7 +28,6 @@ different squads if the workload justifies it.
 
 *Android Leads*  
 Sakis Kaliakoudas • [GitHub](https://github.com/athkalia) • [Website](http://www.sakiskaliakoudas.com)  
-Joao Alves • [GitHub](https://github.com/jcmsalves) • [Twitter](https://twitter.com/jcmsalves) • [Medium](https://medium.com/@jcmsalves)
```
Comment (iTomkinas) [inline README.md:31]: Will I get skyscanner discounts? 😄

**Review by iTomkinas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jcmsalves) [reply README.md:31]: Who knows?

**Review by iTomkinas — APPROVED** [review]

**Comment by athkalia** [comment]
Thanks for everything man, we still love you even though you betrayed us 💣 😅

**Pull request merged**